nibabel/streamlines/tests/test_array_sequence.py

```diff
@@ -0,0 +1,310 @@
+import os
+import unittest
+import tempfile
+import numpy as np
+
+from nose.tools import assert_equal, assert_raises, assert_true
+from nibabel.testing import assert_arrays_equal
+from numpy.testing import assert_array_equal
+from nibabel.externals.six.moves import zip, zip_longest
+
+from ..array_sequence import ArraySequence
+
+
+class TestArraySequence(unittest.TestCase):
+
+    def setUp(self):
+        rng = np.random.RandomState(42)
```

Comment:
But in fact - why not put this as a module-level setup?  You don't need it run before every test do you?

```
SEQ_DATA = {}

def setup():
    global SEQ_DATA
    rng = np.random.RandomState(42)
    SEQ_DATA['rng']  = rng
    SEQ_DATA['data'] = [rng.rand(rng.randint(10, 50), 3) for _ in range(10)]
    ...
```